l10n/README.md

```diff
@@ -269,8 +269,27 @@ python setup.py extract_messages
 
 This will rescan all wrapped text, picking up new strings as well as updating existings strings that have been edited.
 
-_TODO: Github Action to auto-generate messages.pot and fail a PR update if the PR has an out of date messages.pot?_
+### Pre-commit hook for `extract_messages`
 
+A pre-commit hook is configured to run this command automatically before each commit.
+
+To set it up:
+
+```bash
+pip install pre-commit
+pre-commit install
+```
+
+Once installed, the `extract_messages` command will run every time you commit.
```

Comment:
I think naming the pre-commit `extract_messages` creates a minor bit confusion with the built-in `extract_messages` command. I'd prefer a simple rename for the pre-commit to something like `run_extract_messages` or `compare_messages_pot` or `messages_pot_diff_check` etc...